runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -271,34 +288,18 @@ public IClientRouter addClient(IClient client) {
     @SuppressWarnings("unchecked")
     public <T extends IClient> T getClient(Class<T> clientType) {
         return (T) clientList.stream()
-                .filter(clientType::isInstance)
-                .findFirst().get();
+            .filter(clientType::isInstance)
+            .findFirst().get();
     }
 
     @Override
-    public void start() {
-        shutdown = false;
-        if (channel == null || !channel.isOpen()) {
-            Bootstrap b = new Bootstrap();
-            b.group(eventLoopGroup);
-            b.channel(parameters.getSocketType().getChannelClass());
-            parameters.getNettyChannelOptions().forEach(b::option);
-            b.handler(getChannelInitializer());
-            try {
-                connectChannel(b);
-            } catch (SocketException e) {
-                try {
-                    if (shutdownEventLoop) {
-                        eventLoopGroup.shutdownGracefully().sync();
-                    }
-                } catch (InterruptedException ie) {
-                    throw new UnrecoverableCorfuInterruptedError(
-                        "Interrupted while shutting down", ie);
-                }
-                throw new NetworkException(e.getClass().getSimpleName()
-                        + " connecting to endpoint failed", node, e);
-            }
-        }
+    @Deprecated
+    /**
+     * @deprecated The router automatically starts now, so this function call is no
+     *             longer necessary
+     */
+    public synchronized void start() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)